src/pages/using-spark/foundations/color.mdx

```diff
@@ -24,8 +24,11 @@ export const colorsLightSrcSet = `${colorsOnLight} 560w, ${colorsOnLight2x} 1120
 export const colorsDarkSrcSet = `${colorsOnDark} 560w, ${colorsOnDark2x} 1120w, ${colorsOnDark3x} 1680w`;
 
 # Color
-Having a consistent color palette is one of the foundations of a unified user experience.
-These are the colors that have been chosen to represent the brand.
+Our objective is to achieve human-centered design in everything we do.
+
+Having a consistent color palette is one of the foundations of unified user experience.
+
+We take accessibility seriously and comply with WCAG 2.0 and contrast guidelines, and we do our best to account for the needs of all humans.
```

Comment:
I thought I saw a comment from @yodasw16  that said to change 'humans' to people. Did you decide not too?

Reply:
oh, I didn't actually see that comment, but people does sound better.  I was basing it off of the issue. I'll make the change!